Changes

```diff
@@ -2,8 +2,11 @@ Revision history for HTML-FormBuilder
 
```

Comment:
Definitely best to keep the Changes in git since that way we can make sure it is up to date wen we release.  Otherwise it is going to be a maintenance problem.